### DISCUSSION TITLE
Can we use any generic Quasar element inside NiceGUI ?

### DISCUSSION THREAD

**Discussion by Anindya088** [zauberzeug/nicegui#1162] [Q&A] [answered]
### Question

Hi All,

I wanted to check if there is a way to use any generic element from Quasar in NiceGUI? Like for ex: I was reading through the ```Slide Item``` which seemed to me a very nice thing for the swapping effect. 

So I was thinking is there a way to call for any generic Quasar element inside NiceGUI? Please let me know.

Thanks a lot,
Anindya

**Comment by falkoschindler** [comment, marked as answer]
Hi @Anindya088,
Yes, you can basically use any Quasar element using the generic `ui.element`:

```py
with ui.element('q-list').props('bordered separator'):
    with ui.element('q-slide-item') as slide_item:
        with ui.element('q-item'):
            with ui.element('q-item-section'):
                ui.label('Slide me')
        with slide_item.add_slot('left'):
            ui.label('Left')
        slide_item.on('left', lambda: ui.timer(1.0, lambda: slide_item.run_method('reset')))
```

Here is another example: https://github.com/zauberzeug/nicegui/discussions/512#discussioncomment-5314453.

**Reply by Anindya088** [reply]
Hi @falkoschindler , thanks so much for explaining this. I read through the ```ui.element``` on the docs, but was not sure how to use this. This is a great help. Thanks for explaining in so much detail.

Regards,
Anindya
